packages/notification-services-controller/src/NotificationServicesController/__fixtures__/index.ts

```diff
@@ -3,4 +3,3 @@ export * from './mock-notification-trigger';
 export * from './mock-notification-user-storage';
 export * from './mock-raw-notifications';
 export * from './mockResponses';
-export * from './mockServices';
```

Comment:
`mockServices` used `nock`, but was for some reason incompatible with mobile. Removed for now. And we can add back in a later release once we figure out multiple exports and tree shaking.